receiver/mongodbreceiver/testdata/integration/expected.3_0.json

```diff
@@ -13,8 +13,8 @@
                         "dataPoints": [
                            {
                               "asInt": "0",
-                              "startTimeUnixNano": "1659372711430846000",
-                              "timeUnixNano": "1659372771440281000"
+                              "startTimeUnixNano": "1659728796441872000",
```

Comment:
Is this file changed because fixing the partial error problem changed the result?

Reply:
this file does not need to be updated and for this pr